### ISSUE TITLE
metafile doesn't work with multiple entrypoints

### ISSUE THREAD

**Issue by OmgImAlexis** [egoist/tsup#455]
When running this with multiple entrypoints I get a single metafile.

**Comment by egoist** [comment]
yeah, that's how it works,  a metafile contains the information for all the entry points, only multiple formats will create multiple metafiles.
